packages/eslint-config/.prettierignore

```diff
@@ -0,0 +1,5 @@
+**/*.md
```

Comment:
Why they are ignored?

Reply:
I didn't know prettier can handle markdown files....

Reply:
> I didn't know prettier can handle markdown files....

... If it can not, the `ignore` are useless, so all of the following are unnecessary except `CHANGELOG.md`~